test/e2e/apps/cronjob.go

```diff
@@ -105,7 +105,7 @@ var _ = SIGDescribe("CronJob", func() {
 
 		ginkgo.By("Ensuring no jobs are scheduled")
 		err = waitForNoJobs(ctx, f.ClientSet, f.Namespace.Name, cronJob.Name, false)
-		framework.ExpectError(err)
+		gomega.Expect(err).To(gomega.HaveOccurred())
```

Comment:
If this is meant to run into a timeout because the condition for aborting polling is never met, then using `gomega.Consistently` is much better approach.

See also See https://github.com/kubernetes/kubernetes/pull/120301/files#r1363671182.

/lgtm cancel

Reply:
Patrick, thanks your comments. sorry for late reply. also check other conversions. will update together as soon as possible.

Reply:
please review again.

Reply:
@carlory ,please help to review. thanks.